mlrun/db/httpdb.py

```diff
@@ -821,7 +821,8 @@ def list_runs(
     ) -> RunList:
         """
         Retrieve a list of runs, filtered by various options.
-        If no filter is provided, will return runs from the last week.
+        If no filter is provided, it return runs from the last week, partitioned by project/name.
+        To get all runs (without the defaults), specify an explicit time using ``start_time_from``.
```

Comment:
```suggestion
        To get all runs (without the defaults), specify any filter`.
```
I think this sentence may overall be redundant, since the previous one already explains the logic behind the special case of no-filters.